mutiny-wasm/src/indexed_db.rs

```diff
@@ -80,6 +80,29 @@ impl IndexedDbStorage {
         })
     }
 
+    pub(crate) async fn has_mnemonic() -> Result<bool, MutinyError> {
+        let indexed_db = Self::build_indexed_db_database().await?;
+        let tx = indexed_db
+            .transaction(&[WALLET_OBJECT_STORE_NAME], TransactionMode::ReadWrite)
```

Comment:
Does it need to be `ReadWrite` if it's just reading? Same thing for the log checking too.

Reply:
fixed